CMakeLists.txt

```diff
@@ -1,14 +1,15 @@
 cmake_minimum_required(VERSION 3.1)
+project(stc_rpncalc C CXX)
 # 3rd party tools
 find_package(Qt5 COMPONENTS Widgets Qml Quick REQUIRED)
 
+set(CMAKE_BUILD_TYPE "Debug" CACHE STRING "Build type (for tests debug make sense)")
+
 # Compiler warnings
 if(MSVC)
   add_compile_options(/W4 /WX)
 else()
   add_compile_options(-Wall -Wextra -pedantic)
-  set(cpp_flags -std=c++11 -fsanitize=address,undefined)
-  add_compile_options("$<$<COMPILE_LANGUAGE:CXX>:${cxx_flags}>")
 endif()
```

Comment:
This was all copied from another project (not sure if any of this would actually compile under MSVC). Good catch on the address/UB sanitizer, those in fact don't run at all. The UB sanitizer would be nice to have (for all files).